File: app/src/main/kotlin/kotlinstudy/ch02/IsMarriedPersonCreator.java
package kotlinstudy.ch02;

import kotlinstudy.ch02.model.IsMarriedPerson;

public class IsMarriedPersonCreator {
    public static void main(String[] args) {
        IsMarriedPerson sypark = new IsMarriedPerson("sypark", true);
        System.out.println(sypark);
        sypark.setMarried(false);
        System.out.println(sypark);
    }
}

File: app/src/main/kotlin/kotlinstudy/ch02/IsMarriedPersonCreator.kt
package kotlinstudy.ch02

import kotlinstudy.ch02.model.IsMarriedPerson

fun main(){
    val person = IsMarriedPerson("sypark", true)
    println(person.name)
    println(person.isMarried)
    person.isMarried = false
    println(person.isMarried)
}